CHANGELOG.md

```diff
@@ -11,6 +11,9 @@
 
 * **[Fix]** Fix crash involving SDK's `ms_viewWillAppear` method.
 
+### App Center Data
+
+* **[Breaking change]** Rename delegate method `data:didCompletePendingOperation:forDocument:withError:` from `MSRemoteOperationDelegate` to `data:didCompleteRemoteOperation:forDocumentMetadata:withError:`.
```

Comment:
I think it is okay but we haven't used `[Breaking change]`. Is it caused by a bug fix or additional features? Would be better to put more details why we change this **only if it is possible and reasonable**. Otherwise it should be good to keep it.

Reply:
@jaelim-ms 
the change is to match the Android callback, it's not an additional feature:
https://github.com/microsoft/appcenter-sdk-android/blob/4ae6533d2babfc50032e7aed30ab60a29ecb9361/sdk/appcenter-data/src/main/java/com/microsoft/appcenter/data/models/RemoteOperationListener.java#L22